services/galley/src/Galley/Data/Conversation/Types.hs

```diff
@@ -51,7 +51,7 @@ data NewConversation = NewConversation
   { ncType :: ConvType,
     ncCreator :: UserId,
     ncAccess :: [Access],
-    ncAccessRole :: AccessRole,
+    ncAccessRole :: Set AccessRoleV2,
```

Comment:
```suggestion
    ncAccessRoles :: Set AccessRoleV2,
```

Reply:
Ah, maybe you didn't add the `s` because it makes the JSON schema harder?  In that case I don't feel so strongly about this.  [Also see here](https://github.com/wireapp/wire-server/pull/2035#discussion_r786634545)